queries.sql

```diff
@@ -8,3 +8,39 @@ select name, escape_attempts from animals where weight_kg>10.5
 select * from animals where neutered is true
 select * from animals where name != 'Gabumon'
 select * from animals where weight_kg between 10.4  and 17.3
+
+
+UPDATE animals 
+SET species = 'digimon'
+WHERE name like '%mon%' 
+
+UPDATE animals 
+SET species = 'pokemon'
+WHERE species is null
+
```

Comment:
- [ ] @JakariaJishan kindly add a query to verify the change was made, also add a commit to the transaction and then add another query to verify the change persisted after commit
![image](https://user-images.githubusercontent.com/61408860/229922220-03acd5df-a357-4ae3-b390-ea7652a0b981.png)